osmaxx-py/osmaxx/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+from django.test import TestCase
+import os
+from osmaxx.utils import PrivateSystemStorage
+
+
+class PrivateSystemStorageTestCase(TestCase):
+    def setUp(self):
+        random_directory_name_length = 10
+        self.random_directory_name = ''.join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(random_directory_name_length)
+        )
+
+    def test_creates_a_new_directory_if_it_does_not_exist(self):
+        random_directory_path = os.path.join('/tmp/', self.random_directory_name)
```

Comment:
What's the benefit of testing this with a random directory name instead of a hard-coded directory name? I'd get if you want to discover unforeseen edge cases and boundary conditions, but a lower-case alphanumeric name is unlikely to hit those, however random it might be.